editor/modes/visual-editor/block.js

```diff
@@ -167,9 +170,9 @@ class VisualEditorBlock extends Component {
 	}
 
 	maybeHover() {
-		const { isHovered, isSelected, isMultiSelected, onHover } = this.props;
+		const { isHovered, isSelected, isMultiSelected, isInserterOpen, onHover } = this.props;
 
-		if ( isHovered || isSelected || isMultiSelected ) {
+		if ( isHovered || isSelected || isMultiSelected || isInserterOpen ) {
```

Comment:
I guess this check is to avoid "removing" the inserter when we hover out of the block? 
Maybe we can handle this in a simpler way: select the block when we open its inserter.
I'm just trying to avoid this dependency between the inserter and this component.

Reply:
>Maybe we can handle this in a simpler way: select the block when we open its inserter.

That sounds like a good idea! I'll give it a try.